internal/infrastructure/ceph/rbd.go

```diff
@@ -4,22 +4,33 @@ import (
 	"bufio"
 	"encoding/binary"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"math"
 	"os"
 	"strconv"
 	"unsafe"
 
+	"github.com/cespare/xxhash/v2"
 	"github.com/cybozu-go/fin/internal/model"
+	"github.com/cybozu-go/fin/internal/pkg/csumio"
 	"github.com/cybozu-go/fin/internal/pkg/zeroreader"
 	"golang.org/x/sys/unix"
 )
 
 var (
 	DefaultExpansionUnitSize uint64 = 100 * 1024 * 1024 * 1024 // 100 GiB
+	ErrChecksumMismatch             = csumio.ErrChecksumMismatch
```

Comment:
I intended to define a completely new error here. In my opinion, `csumio.ErrChecksumMismatch` is a detail of the internal implementation, and the user of `package ceph` doesn't need to know it.


```suggestion
	ErrChecksumMismatch             = errors.New("checksum mismatch")
```